themes/nav-community-v1/layouts/partials/footer.html

```diff
@@ -36,7 +36,8 @@ <h3>Important Links</h3>
                         <ul>
                             <li><a href="https://www.navcoin.org" target="_blank">NavCoin.org</a></li>
                             <li><a href="https://github.com/NAVCoin">GitHub</a></li>
-                            <li><a href="https://chainz.cryptoid.info/nav/">Block Explorer</a></li>
+                            <li><a href="https://chainz.cryptoid.info/nav/">Block Explorer - Cryptoid</a></li>
+                            <li><a href="https://www.navexplorer.com">Block Explorer - NavExplorer</a></li>
```

Comment:
maybe Cryptoid should be cryptoID? that's how they have it on their site / logo.